test/test_error.c

```diff
@@ -251,9 +298,20 @@ int main(int argc, char **argv)
     if (0 > H5Fclose(fid)) ERROR(H5Fclose);
 
     /* Use raw file I/O to corrupt the dataset named corrupted_data */
+
+#if defined(_WIN32)
+    FILE *fp;
+    errno_t err  = fopen_s( &fp, FNAME, "rb+");
+    if(err != 0) ERROR(fopen_s);
+    fseek(fp, (off_t) off + (off_t) siz / 3, SEEK_SET);
+    fwrite(corrupt, 1 , sizeof(corrupt), fp);
+    fclose(fp);
+#else
+    int fd;
```

Comment:
I don't think we really *need* to use `pwrite` here. We can just replace entirely with logic specified above which I believe should work also on Linux/macOS

Reply:
Sounds good, I'll update it.

Reply:
See PR #112